sdk/core/Azure.Core/perf/ResourceIdentifierBenchmark.cs

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+using BenchmarkDotNet.Attributes;
+
+namespace Azure.Template.Perf
```

Comment:
nit: namespace

@pakrym, is this our standard approach for including benchmarks in CI?  Wondering for my own reference.

Reply:
CI won't run actual benchmarks but will keep the code buildable.